docs/en/docs/tutorial/path-params.md

```diff
@@ -222,7 +222,7 @@ Nevertheless, you can still do it in **FastAPI**, using one of the internal tool
 
 And the docs would still work, although not adding any documentation telling that the parameter should contain a path.
 
-### Path convertor
+### Path converter
```

Comment:
Here as well, we should stick to `convertor`, to be consistent with the code base.